src/otx/recipe/detection/rtmdet_tiny.yaml

```diff
@@ -0,0 +1,151 @@
+model:
+  class_path: otx.algo.detection.rtmdet.RTMDetTiny
+  init_args:
+    label_info: 80
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.0007
```

Comment:
how this number was chosen?

Reply:
I used HPO with optimizer (SGD, AdamW) and learning rate on 4 benchmark datasets (bdd medium/large, pothole medium, vitens large).
I got 8 hyper parameter candidates and 1 official hyper parameter, I did additional benchmark experiments to finalize the hyper parameter.
This number showed the best performance with shorter epochs.